r1cs/gadgets/crypto/src/prf/ripemd160.rs

```diff
@@ -304,6 +319,10 @@ mod test {
     use rand_xorshift::XorShiftRng;
 
     #[test]
+    /// Tests satisfiability of the circuit for the compression function, choosing a random
+    /// input block.
+    // TODO: What is the use of this test? We do not even match the output against the
```

Comment:
We just check that all the constraint passes, no errors happen and that the number of constraints is the expected one